modules/admin/proc-customizing-telemetry-segment-using-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id="proc-customizing-telemetry-segment-using-operator_{context}"]
+= Customizing telemetry Segment source using the Operator
+
+You can configure integration with your Segment source by using the Operator.
+
+.Procedure
+
+. Add the following YAML code in your `Backstage` custom resource (CR):
++
+[source,yaml]
+----
+# ...
+extraEnvs:
+  envs:
+    - name: SEGMENT_WRITE_KEY
+      value: <segment_key> # <1>
```

Comment:
This is the proper definition. 
```suggestion
spec:
  application:
    extraEnvs:
      envs:
        - name: SEGMENT_WRITE_KEY
          value: <segment_key> # <1>
```

It is accessible also by `Form view`: application -> extraEnvs -> envs
<img width="564" alt="image" src="https://github.com/user-attachments/assets/58062309-01ed-4601-9363-3cc21b234d1c">